snippets/go.json

```diff
@@ -155,6 +155,18 @@
 		"test function": {
 			"prefix": "tf",
 			"body": "func Test$1(t *testing.T) {\n\t$0\n}"
+		},
+		"init function": {
+			"prefix": "finit",
```

Comment:
Consistency does make sense and drive usability as well. Lets see if there is any other feedback over the weekend. I'll merge the PR (`fmain` and `finit`) on Monday if we don't hear anything more.